server/db/migrate/20220408032834_change_gene_id_field.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeGeneIdField < ActiveRecord::Migration[6.1]
+  def self.up
+    rename_column :genes, :entrez_id, :concept_id
+  end
+
+  def self.down
+    rename_column :genes, :concept_id, :entrez_id
+  end
+end
```

Comment:
does up and down correspond to the jump from "claim" to normalized record status?

Reply:
This is actually Ruby migration lingo, the idea being that when you write a structural change to the DB (up) you can also write a routine to undo it (down)